boards/nucleo32-common/include/periph_conf_nucleo32.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C)  2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nucleo-common
```

Comment:
shouldn't it be boards_nucleo32-common ? (not sure)

Reply:
yes it should (just checked)

Reply:
fixed